packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/VideoController/VideoController.js

```diff
@@ -0,0 +1,95 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Box, Button, Select } from 'grommet'
+import { CirclePlay, PauseFill } from 'grommet-icons'
+import FormattedTime from './FormattedTime'
+import Slider from './Slider'
+
+const VideoController = ({
+  isPlaying,
+  onPlayPause,
+  onSpeedChange,
+  played,
+  playbackRate,
+  duration,
+  onSliderMouseUp,
+  onSliderMouseDown,
+  onSliderChange
+}) => {
+  return (
+    <Box
+      background={{
+        dark: 'dark-3',
+        light: 'light-3'
+      }}
+    >
+      <Box pad={{ horizontal: 'xsmall', top: 'xsmall' }}>
+        <Slider
+          played={played}
+          onMouseUp={onSliderMouseUp}
+          onMouseDown={onSliderMouseDown}
+          onChange={onSliderChange}
+        />
+      </Box>
+
+      <Box direction='row' justify='between'>
+        <Box direction='row'>
+          <Box
+            a11yTitle='Video Play and Pause button'
+            alignSelf='center'
```

Comment:
Similar comment to before. I think you want to label the button here, not its container. Since Play/Pause is a toggle, then I think the current state (playing or paused) needs to be announced in the accessibility tree too. Here's a great [overview of how play/pause state is announced](https://sarahmhigley.com/writing/playing-with-state/) in various screen reader/browser combinations. It seems you like you need a combination of `aria-label` and `aria-pressed` on the button.